examples/cloudflare-workers/README.md

```diff
@@ -0,0 +1,31 @@
+<head>
+  <meta name="Momento Node.js Client Library Documentation" content="Node.js client software development kit for Momento Cache">
+</head>
+<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+
+# Using Momento inside a Cloudflare worker
+
+## Example Requirements
+
+- Node version 14 or higher is required
+- A Momento Auth Token is required, you can generate one using the [Momento Console](https://console.gomomento.com)
+
+## Examples
+
+This directory contains subdirectories for two types of example projects to use Momento inside a Cloudflare worker:
+
+[HTTP-API](./http-api) - How to interact with Momento cache using its HTTP API on Cloudflare workers.
+[Web-SDK](./web-sdk) - How to interact with Momento cache using its web SDK on Cloudflare workers.
+
+There are a few reasons why you might choose one over the other when interacting with Momento within a Cloudflare Worker.
```

Comment:
@cprice404 please review this section.. added a few things based on my understanding and the generic pros/cons that i read upon